### PR TITLE
Introduce simulation generators

### PR DESCRIPTION
**Pull request by fpagnoux**
Introduce a helper function to generate arbitrarily big populations. 

I used it a lot to investigate performances, and it could be reused. I put in in tests, but there may be a better spot.

This is clearly low priority

### CODE REVIEW

File: tests/core/simulation_generator.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    """
+        Generate a simulation containing nb_persons persons spread in nb_groups groups
+    """
+    simulation = Simulation(tax_benefit_system = tax_benefit_system, **kwargs)
+    simulation.persons.ids = np.arange(nb_persons)
+    simulation.persons.count = nb_persons
+    adults = [0] + sorted(random.sample(xrange(1, nb_persons), nb_groups - 1))
+
+    members_entity_id = np.empty(nb_persons, dtype = int)
+    members_legacy_role = np.empty(nb_persons, dtype = int)
```
Comment (Anna-Livia) [inline tests/core/simulation_generator.py]: Would it be helpful to write a reminder of the `legacy_role` indexes ?

File: tests/core/simulation_generator.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import numpy as np
+
+import random
+from openfisca_core.simulations import Simulation
+from openfisca_core.periods import period as make_period
+
+
+def make_simulation(tax_benefit_system, nb_persons, nb_groups, **kwargs):
+    """
+        Generate a simulation containing nb_persons persons spread in nb_groups groups
```
Comment (Anna-Livia) [inline tests/core/simulation_generator.py]: Would an example (or a link to one) would be useful for fist time users ?

File: tests/core/simulation_generator.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            entity.members_entity_id = members_entity_id
+            entity.members_legacy_role = members_legacy_role
+            entity.count = nb_groups
+            entity.members_role = np.where(members_legacy_role == 0, entity.flattened_roles[0], entity.flattened_roles[-1])
+    return simulation
+
+
+def randomly_init_variable(simulation, variable_name, period, max_value, condition = None):
+    """
+        Initialise a variable with random values (from 0 to max_value) for the given period.
```
Comment (Anna-Livia) [inline tests/core/simulation_generator.py]: Would an example (or a link to one) would be useful for fist time users ?

**Comment by Anna-Livia** [comment]
As for the directory, would openfisca_core/tools/script be more appropriate ?

File: tests/core/simulation_generator.py (hunk 1 above)
Comment (fpagnoux) [reply tests/core/simulation_generator.py]: Added a comment to explain what legacy roles are

File: tests/core/simulation_generator.py (hunk 2 above)
Comment (fpagnoux) [reply tests/core/simulation_generator.py]: Added an example

File: tests/core/simulation_generator.py (hunk 3 above)
Comment (fpagnoux) [reply tests/core/simulation_generator.py]: Added an example

**Pull request merged**